### DISCUSSION TITLE
GazeboDrone make error

### DISCUSSION THREAD

**Discussion by chongjeelee** [microsoft/AirSim#4223] [General]
when i make GazeboDrone ,error occured here.
![图片](https://user-images.githubusercontent.com/30140969/146170700-8b21ad17-3755-4df5-b791-09c8c7242cd4.png)
----------
someone knew how to solve it ?thanks.

**Comment by jonyMarino** [comment]
@chongjeelee it seems that you didn't compile airsim with the --gcc option
